Registry/Registry.EM_COMMON

```diff
@@ -965,6 +965,9 @@ state   real   QSFC             ij     misc        1         -      r        "QS
 state   real   AKHS             ij     misc        1         -      r        "AKHS"                  "SFC EXCH COEFF FOR HEAT"                      "m s-1"    
 state   real   AKMS             ij     misc        1         -      r        "AKMS"                  "SFC EXCH COEFF FOR MOMENTUM"                  "m s-1"    
 state   integer KPBL            ij     misc        1         -     r         "KPBL"                  "LEVEL OF PBL TOP"                             ""
+state   real   U10E             ij     misc        1         -     irhdu     "U10E"                  "Special U at 10 M from MYJSFC"         "m s-1"
```

Comment:
@weiwangncar 
Wei,
We can address this now or later. For a diagnostic computed in the MYJ SFC scheme:
1. We should not need "i" - this is not an input from the real program (no mods to module_intialize_real)
2. We should not need "r" - this diagnostic should have no impact on restarts
3. We should not need "d" - as a diagnostic, we do not need to get an initial value from the parent domain

To be fair, these are all on the U10/V10 fields. So, we can address this now or later.

Reply:
@davegill Agreed that i and r may not be needed. but d may be if the option is used with moving nests.

Reply:
@weiwangncar 
Wei,
Let's leave the "d" and check a moving nest case when we have more time. The "d" nesting option _should_ be unnecessary for a physics diagnostic.